### ISSUE TITLE
Ch3 Notebook: Random Forest Model

### ISSUE THREAD

**Issue by Jackal08** [BlackArbsCEO/Adv_Fin_ML_Exercises#23]
In the chapter 3 notebook, I noticed that when the train test split function is as follows: ```X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5)``` This function shuffles the data by default. I believe this is a problem because it introduces look ahead bias into the time series (Maybe we could consider it a light type of label leakage). This is also the reason why purge k fold and embargo CV is introduced in later chapters. 

To correct this the code needs to read ```X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, shuffle=False)```

**Comment by BlackArbsCEO** [comment]
Nice catch. Updated the notebooks. Fixed dd47196ae766c385b4a6ffe1519c39c89b740851
